bom/application/pom.xml

```diff
@@ -90,7 +91,7 @@
         <commons-lang3.version>3.12.0</commons-lang3.version>
         <commons-codec.version>1.15</commons-codec.version>
         <classmate.version>1.5.1</classmate.version>
-        <hibernate-orm.version>5.6.5.Final</hibernate-orm.version>
+        <hibernate-orm.version>5.6.5.Final</hibernate-orm.version> <!-- When updating, align bytebuddy.version to Hibernate needs as well -->
```

Comment:
Usually, in this case, we don't respect the alphabetical order and we add the related dependency just below. But I don't really mind :).

Reply:
ok I'll gladly change that as I'd prefer them close as well.  I had (wrongly) assumed that people wouldn't have liked that :)